app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -43,65 +42,57 @@ const Notes = () => {
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
-          />
+    <div className="flex w-full">
+      <Menu showMenu={showMenu} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(oldShowState => !oldShowState)}
```

Comment:
Too much verbose of a variable name. We can make it compact by naming it `showMenu`. Whether it's old or not can be implied by the context of its usage.

Reply:
Ok, I will rename it to `showMenu`.